packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,75 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleObject } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleObject<typeof parts>> = {
+  sm: {
+    label: { textStyle: 'text-body-meta-regular' },
```

Comment:
I believe `textStyle` isn't applying the right styling to the component. Could you check how this is done in the `Badge` component?

Reply:
`textStyle` is the prop suggested by chakra in order to deal with font-size, line-height [chakra-ui](https://chakra-ui.com/docs/features/text-and-layer-styles#text-styles). It is used within `Button` component too.

Reply:
This is a known bug in chakra and we've solved this issue in some components as the `Button` and `Badge`. Please check these components to see how's done now.

If you check your radio styles you'll see that textStyles are not being applied correctly.

Reply:
I can not sse within the `Button` prop a different prop to set fonts, I am seeing the same `textStyle` prop that I am using here and checking the CSS props within Radio are set right. 
<img width="356" alt="image" src="https://user-images.githubusercontent.com/42104272/150179036-730d426c-2a17-4d8e-92bb-d45d4a4ce0f1.png">

Reply:
Do you have the latest changes from `master`? Notice that `Button` uses the theme parameter...
`    xs: ({ theme }) => ({
      ...theme.textStyles['text-body-floating-label-medium'],
      px: '0.5rem',
      py: '0.25rem',
    }),`

Reply:
my bad. I am working in this branch based  on `with-chakra` branch and in that branch there are not the last updates from master. I updated the code using this new approach